R/auto.R

```diff
@@ -173,7 +173,7 @@ get_news_headers <- function() {
   news <- readLines(news_path)
   top_level_headers <- grep("^# [a-zA-Z][a-zA-Z0-9.]*[a-zA-Z0-9] [0-9.-]+", news)
   top_level_headers <- utils::head(top_level_headers, 2)
-  gsub("^# [^0-9]+", "", news[top_level_headers])
+  gsub("^# .* ", "", news[top_level_headers])
```

Comment:
What problem does this solve? If it's relevant in the main branch already, we should fix there (via pull request) and merge into this branch.